python/stressng/README.md

```diff
@@ -0,0 +1,71 @@
+# Official plugins repository for Arcaflow
+
```

Comment:
should this readme be inside the stressng folder?

Reply:
that's where it is for other plugins. Not sure though.

Reply:
@janosdebugs What's your take?

Reply:
Yes -- Please update the readme to provide plugin-specific description and use.

Reply:
done